ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,130 @@
+
+name: Monorepo labels
+description: Import on 2023-05-09
+
+codes:
+  - code: A
+    name: Action 
+    description: used with GHA and trigger a certain process
+  - code: C
+    name: Complexity 
+    description: Complexity of the issues, also used for contribution guidelines.
+  - code: I
+    name: Issues 
+    description: Categorisation of issues
+  - code: T
+    name: Topics
+    description: PR/Issue is related to this topic
+
+labels:
+  - name: A0-needs_burnin
+    description: Pull request needs to be tested on a live validator node before merge. DevOps team is notified via matrix.
+    color: c7def8
+  - name: A1-insubstantial
+    description: Pull request requires no code review (e.g., a sub-repository hash update).
+    color: C6C7F8
+  - name: A2-stale
+    description: Pull request did not receive any updates in a long time. No review needed at this stage. Close it.
+    color: F2C5F8
+  - name: A3-backport
+    description: Pull request is already reviewed well in another branch.
+    color: F8C5E7
+  - name: C0-easy
+    description: Can be fixed primarily by duplicating and adapting code by an intermediate coder.
+    color: eeeeee
+  - name: C1-medium
+    description: Can be fixed by a coder with good Rust knowledge but little knowledge of the codebase.
+    color: eebb00
+  - name: C2-substantial
+    description: Can be fixed by an experienced coder with a working knowledge of the codebase.
+    color: D4C5F9
+  - name: C3-involved
+    description: Can be fixed by an expert coder with good knowledge of the codebase.
+    color: E99695
+  - name: C4-mentor
+    description: An easy task where a mentor is available. Please indicate in the issue who the mentor could be.
```

Comment:
We could decouple the `mentor` tag from the Complexity of the task, there could be non-easy tasks that can be mentored. Not sure where the `mentor` label could land if we remove it from Complexity though.
cc @sam0x17

Reply:
I mean yes, you could mentor a lot of things, but you do this any way all the time for your issues. However, it is questionable if you want to "pull" people for these tasks. Mentor tasks should be for new people and then later they can take on more complicated tasks, which do not mean that you will not help them.

Reply:
I agree @bkchr but that's why I think it would be useful -- in that rare scenario where there is something hard/involved that the subject matter expert is too busy to work on directly, but they would be happy for someone to pick it up and they will guide them through it a little bit, they can apply the mentor label along with a non-easy label

Reply:
In other words, I think we should at least acknowledge the possibility of advanced mentoring, because we'd definitely all benefit from a bit more of that going around

Reply:
This might for example give someone more confidence that "hey, I know what I'm doing but I don't know this particular area very well, but I see they are willing to mentor whoever picks it up a little bit on it, so maybe I'll pick up this issue and try it" instead of just never attempting it

Reply:
Independently of that, `mentor` isn't a complexity, it doesn't seem logical to leave it in that category

Reply:
the category was called "Contribution labels" before, which we could re-name back to it and leave mentor in it and just change the description so that you can combine also other complexity levels with the mentor label.